donkeycar/parts/telemetry.py

```diff
@@ -15,31 +14,40 @@
 from paho.mqtt.client import Client as MQTTClient
 
 
-class Telemetry(StreamHandler):
-    '''
+class MqttTelemetry(StreamHandler):
+    """
     Telemetry class collects telemetry from different parts of the system and periodically sends updated to the server.
     Telemetry reports are timestamped and stored in memory until it is pushed to the server
-    '''
+    """
 
-    def __init__(self, cfg, default_inputs = [], default_types = []):
+    def __init__(self, cfg, default_inputs=None, default_types=None):
 
         StreamHandler.__init__(self)
 
         self.PUBLISH_PERIOD = cfg.TELEMETRY_PUBLISH_PERIOD
         self._last_publish = time.time()
         self._telem_q = queue.Queue()
-
-        self._default_inputs = default_inputs
-        self._default_types = default_types
+        self._default_inputs = default_inputs if default_inputs is not None else []
```

Comment:
You can rewrite this as `default_inputs or []`.

Reply:
fixed (new to me :)